src/api/environment.cc

```diff
@@ -314,6 +314,10 @@ MultiIsolatePlatform* GetMainThreadMultiIsolatePlatform() {
   return per_process::v8_platform.Platform();
 }
 
+void SetMainThreadMultiIsolatePlatform(MultiIsolatePlatform* platform) {
+  per_process::v8_platform.SetPlatform(static_cast<node::NodePlatform*>(platform));
```

Comment:
Style: long line. `make test` or `make lint` should have complained.

Substance: unsound static_cast. `NodePlatform` inherits from `MultiIsolatePlatform`, not the other way around.

Reply:
@bnoordhuis is there a best practice approach for this? alternatively, i could add an extra version of `CreateNodePlatform` that returned a `NodePlatform*`, which wouldn't change much since the existing `CreateNodePlatform` just downcasts a `NodePlatform`, but would love some guidance if you think there's a another better way!